duckduckgo/duckduckgo-autofill
# Fix new release

**Reviewer:** @shakyShane 
**Asana:** https://app.asana.com/0/0/1203285349078972/f

## Description
This is a follow-up of https://github.com/duckduckgo/duckduckgo-autofill/pull/205. It mostly contains path fixes and such little things. I have also moved Android to use `npm install` so it's more robust and added the extensions repo.

## Steps to test
Tested with a bunch of test releases, spamming half the company 🙂. I've also added a test for the function that creates the PR templates to make sure different platforms get the appropriate content. At this time I wouldn't know what else to test because we're running a bunch of simple `string.replace`. If we encounter weird edge cases we can add more tests. Failures in the flow are pretty harmless, in that they just stop the release automation. We can resume it manually or restart it if needed.
